internal/impl/tmpl/devcontainerDockerfile.tmpl

```diff
@@ -27,4 +27,5 @@ WORKDIR /code
 RUN sudo chown $DEVBOX_USER:root /code
 COPY devbox.json devbox.json
 RUN devbox install
+RUN devbox shellenv --init-hook >> ~/.profile
 CMD ["devbox", "shell"]
```

Comment:
This line seems redundant

Reply:
@apgrucza true. But the problem is we use this template file for both devcontainer setup and for general docker setup. 
I think the best way is to remove the CMD and keep shellenv for devcontainer and vice versa for `devbox generate dockerfile`.

Reply:
Ok, but maybe the same change would be beneficial for the general Dockerfile too?